connection_plugins/machinectl.py

```diff
@@ -38,7 +38,7 @@
 from ansible.errors import AnsibleError
 from ansible.plugins.connection import ConnectionBase
 from ansible.utils.vars import merge_hash
-from ansible.utils.unicode import to_bytes
+from ansible.module_utils._text import to_bytes
```

Comment:
I believe that's typically what a leading underscore indicates, but a lot of stuff in Ansible plugin land starts with leading underscores even though there's basically no way to avoid using it (esp. the `_task` attribute for action plugins).  I think it's probably fine.